packer/README.md

```diff
@@ -34,6 +34,58 @@ vagrant box remove centos7-katello-devel-stable
 sudo virsh vol-delete --pool default centos7-katello-devel-stable_vagrant_box_image_0.img
 ```
 
+### Creating version specific dev boxes
```

Comment:
Right now this is specific to Katello, I think we should make that clear, i.e. `Creating version specific Katello dev boxes`. Also, a little bit of explanation about what the box is would be nice, i.e. `This process builds a stable development environment from release branches that can be used for support and debugging purposes`

Reply:
We should point out packer needs to be installed and available, i know it's above, but many will just be reading this section and the existing non-vagrant `packer` package command will just hang when the command runs.

Centos7 and fedora come with an existing binary called 'packer', so maybe the script should use `/usr/bin/packer`? That is where the rpm installs to. We can point this out as a pre-requisite too.

Reply:
good point, updating the script to use /usr/bin/packer and adding clarifying text